### ISSUE TITLE
Set restrictions with number inequalities don't work for fractions

### ISSUE THREAD

**Issue by benruijl** [vermaseren/form#75]
The following pattern does not match, even though comparators <,>,== can be applied to combinations of integers and fractions:

	S n;
	CF K;

	L F =  K(3/2);
	id K(n?{>1}) = 1;

	Print +s;
	.end

**Comment by tueda** [comment]
Empirically I know that `K(n?{>1})` matches with *small* integers greater than 1 (so doesn't match with `K(10^10)`), but actually where can we find the description of it in the manual, except some explanations in the tutorial?

**Comment by jodavies** [comment]
On my system the match fails precisely at K(2^31), i.e., for numbers above the largest signed 32bit int.

**Comment by tueda** [comment]
Yes, by *small* I meant a small integer that fits in a WORD, so 0 <= |n| < 2^31 on 64-bit systems. (One bit is used for the sign.)
